internal/controller/server/execution/registration/execution_test.go

```diff
@@ -2,341 +2,36 @@ package registration
-	jobRepoMock := new(job.JobMock)
-	metadataRepoMock := new(metadata.MetadataMock)
-	mockScheduler := new(scheduler.SchedulerMock)
-	mockRandomIdGenerator := new(idgen.IdGeneratorMock)
+func TestRegistration(t *testing.T) {
 	executorRepoMock := new(executor.ExecutorMock)
+	testExec := NewRegistrationExec(executorRepoMock)
 
-	testExec := NewExec(metadataRepoMock, executorRepoMock, jobRepoMock, mockRandomIdGenerator, mockScheduler)
-
-	testExecutionData := &clientCPproto.RequestForExecute{
-		JobName: "testJobName1",
-		JobData: map[string]string{
-			"env1": "envValue1",
+	testRegisterRequest := protofiles.RegisterRequest{
+		ID: "Random-Id",
+		ExecutorInfo: &protofiles.ExecutorInfo{
+			Info: "Test Executor",
 		},
 	}
-	testJobID := uint64(12345)
-
-	jobRepoMock.On("CheckJobIsAvailable", testExecutionData.JobName).Return(true, nil)
-	jobRepoMock.On("ValidateJob", testExecutionData).Return(true, nil)
-	mockRandomIdGenerator.On("Generate").Return(testJobID, nil)
-	mockScheduler.On("AddToPendingList", testJobID, testExecutionData).Return(errors.New("failed to add job in pending list in scheduler"))
-
-	jobId, err := testExec.ExecuteJob(context.Background(), testExecutionData)
-	assert.Equal(t, "failed to add job in pending list in scheduler", err.Error())
-	assert.Equal(t, uint64(0), jobId)
-	jobRepoMock.AssertExpectations(t)
-	mockScheduler.AssertExpectations(t)
-	mockRandomIdGenerator.AssertExpectations(t)
-}
-
-func TestGetMetadata(t *testing.T) {
-	jobRepoMock := new(job.JobMock)
-	metadataRepoMock := new(metadata.MetadataMock)
-	mockScheduler := new(scheduler.SchedulerMock)
-	mockRandomIdGenerator := new(idgen.IdGeneratorMock)
-	executorRepoMock := new(executor.ExecutorMock)
 
-	testExec := NewExec(metadataRepoMock, executorRepoMock, jobRepoMock, mockRandomIdGenerator, mockScheduler)
-	testClientInfo := &clientCPproto.ClientInfo{
-		ClientEmail: "test@gmail.com",
-		AccessToken: "random",
+	testRegisterResponse := protofiles.RegisterResponse{
+		Registered: true,
 	}
-	testRequestForDescribe := &clientCPproto.RequestForDescribe{
-		JobName:    "testJobName",
-		ClientInfo: testClientInfo,
-	}
-	var testMetadata = &clientCPproto.Metadata{
-		Name:        "testJobName",
-		Description: "This is a test image",
-		ImageName:   "images/test-image",
-	}
-	metadataRepoMock.On("GetValue", testRequestForDescribe.JobName).Return(testMetadata, nil)
-	resultMetadata, getMetadataErr := testExec.GetMetadata(context.Background(), testRequestForDescribe)
-	assert.Equal(t, testMetadata, resultMetadata)
-	assert.Nil(t, getMetadataErr)
-	metadataRepoMock.AssertExpectations(t)
-
-}
-
-func TestGetJobList(t *testing.T) {
-	jobRepoMock := new(job.JobMock)
-	metadataRepoMock := new(metadata.MetadataMock)
-	mockScheduler := new(scheduler.SchedulerMock)
-	mockRandomIdGenerator := new(idgen.IdGeneratorMock)
-	executorRepoMock := new(executor.ExecutorMock)
-
-	testExec := NewExec(metadataRepoMock, executorRepoMock, jobRepoMock, mockRandomIdGenerator, mockScheduler)
-
```

Comment:
These tests are removed from here but I cant see where they have been replaced.

Reply:
None of the tests are removed completely. It is not reflected because primarily all tests were copied from old code so we removed multiple instances of tests.